doc_admin/AdminManual.adoc

```diff
@@ -1492,7 +1486,8 @@ dicom.storage.filesystem=Z:/OH/Shared/Dicom_images
 dicom.max.size=4M
 ----
 
-NOTE: The _dicom.properties_ file is automatically generated/overwritten at any **oh.sh / oh.ps1**  startup, unless MANUAL_CONFIG option is set to "on" in the scripts.
+NOTE: The _dicom.properties_ file can be automatically generated/overwritten at any **oh.sh / oh.ps1**  startup, by setting GENERATE_CONFIG_FILES option is set to "on" in the **oh.sh / oh.ps1** scripts.
```

Comment:
I would add "the" after "setting" here too

Reply:
Done. thanks !